### PR TITLE
Closes #74 - Add unit tests for Task controller.

### LINKED ISSUE #74: Add unit tests for Task controller.

### PR DESCRIPTION
**Pull request by YarkoKhamar**
I've added unit tests for the Tasks controller I've implemented previously. Also, I refactored the code in Tasks service to unify formatting and follow some established code styles.

### CODE REVIEW

File: ServiceTier/Services/TaskService.cs [hunk 1]
```diff
@@ -82,15 +83,19 @@ public void UpdateTask(TaskDTO taskDTO)
             unitOfWork.Save();
         }
 
-        public void UpdateTaskStatus(int taskId, bool newStatus)
+        public void UpdateTaskStatus(TaskDTO model)
         {
-            Tasks task = unitOfWork.Tasks.GetByID(taskId);
-            if (task == null)
+            if (model.IsRunning)
```
Comment (roman-fay) [inline ServiceTier/Services/TaskService.cs]: so basically model.IsRunning = !model.IsRunning? why are we flipping this value?

Comment (YarkoKhamar) [reply ServiceTier/Services/TaskService.cs]: Yes, I will change to model.IsActive = !model.IsActive; Also, IsRunning was a mistake by me. I meant to flip the status of task to archive it and make active again.

Comment (roman-fay) [reply ServiceTier/Services/TaskService.cs]: still not sure why we are just flipping that flag on every update. can you explain?

Comment (YarkoKhamar) [reply ServiceTier/Services/TaskService.cs]: Well I had envisioned that on our tasks page only active tasks will be listed. Each task will have the button to archive that task . That button will trigger this UpdateStatus() method and IsActive property of underlying task will be changed to false. Then the task will be displayed only on our archive page.

Comment (YarkoKhamar) [reply ServiceTier/Services/TaskService.cs]: And vice versa when making archived task active again.

**Pull request merged**